homeassistant/components/tessie/strings.json

```diff
@@ -87,6 +87,62 @@
       "climate_state-passenger_temp_setting": {
         "name": "Passenger Temperature Setting"
       }
+    },
+    "binary_sensor": {
+      "charge_state-battery_heater_on": {
+        "name": "Battery Heater"
```

Comment:
Please keep the names in Sentence case instead of Title Case

Reply:
Add _new_ strings have been renamed.